src/subscription_manager/repolib.py

```diff
@@ -23,13 +23,19 @@
 from subscription_manager import model
 from subscription_manager.model import ent_cert
 from subscription_manager.repofile import Repo, manage_repos_enabled, get_repo_file_classes
-from subscription_manager.repofile import YumRepoFile
+from subscription_manager.repofile import YumRepoFile, RepoFileBase
+from subscription_manager.repofile import HAS_DEB822, HAS_ZYPP
 from subscription_manager.utils import get_supported_resources
 
 import rhsm.config
 import configparser
 from rhsmlib.facts.hwprobe import HardwareCollector
 
+if HAS_DEB822:
+    from rhsm.repofile import AptRepoFile
```

Comment:
It is wrong name of Python module. It should be `from subscription_manager.repofile import AptRepoFile`.